internal/locald/rootmanager/sbmgr_monitor.go

```diff
@@ -193,7 +193,6 @@ func (mon *sbmgrMonitor) stop() error {
 }
 
 // checkStatusLoop periodically checks sandbox manager status while it's running
-// If session is released, stops the monitor to prevent restart
 func (mon *sbmgrMonitor) checkStatusLoop(procDone <-chan struct{}) {
```

Comment:
We can get rid of this logic, it's not doing anything:

https://github.com/signadot/cli/blob/5092d56ccadff00185ed0a93b1833bf5b0a2ab01/internal/locald/rootmanager/sbmgr_monitor.go#L195-L212